sql/core/src/test/resources/sql-tests/inputs/table-valued-functions.sql

```diff
@@ -24,3 +24,7 @@ select * from RaNgE(2);
 
 -- Explain
 EXPLAIN select * from RaNgE(2);
+
+-- cross-join table valued functions
+SET spark.sql.crossJoin.enabled=true;
```

Comment:
Could you remove this line? If we specify CROSS JOIN in the query, no need to set this parm.